README.md

```diff
@@ -14,6 +14,20 @@ Open [http://localhost:4000](http://localhost:4000) to view it in your browser.
 The page will reload when you make changes.\
 You may also see any lint errors in the console.
 
+### API
+
+This project makes use of the [`json-server` npm package](https://github.com/typicode/json-server) to serve the included `data.json` file as an API. To run the API, install `json-server` globally:
+
+```
+npm install -g json-server
+```
+
+Then start the API:
+
+```
+json-server --watch src/data.json
+```
+
```

Comment:
Just a bit of extra documentation in the README to help people get the app up and running.